### PR TITLE
update styles from home page [WIP]

### PR DESCRIPTION
**Pull request by hectoruch**
update styles from home page

### CODE REVIEW

File: app/styles/components/c-box-triangle.scss [hunk 1]
```diff
@@ -1,31 +1,31 @@
...
-  transform: rotate(-135deg);
-  -webkit-transform: rotate(-135deg);
+.c-box-triangle {
   -moz-transform: rotate(-135deg);
-  -o-transform: rotate(-135deg);
   -ms-transform: rotate(-135deg);
-  border-right: 2px solid $color9;
-  border-bottom: 2px solid $color9;
-  margin-bottom: -100px;
+  -o-transform: rotate(-135deg);
```
Comment (clementprdhomme) [inline app/styles/components/c-box-triangle.scss]: Useless: remove all the prefixes, `autoprefixer` will do it for us.

`transform: rotate(-135deg);`

File: app/styles/components/c-box-triangle.scss [hunk 2]
```diff
@@ -1,31 +1,31 @@
...
+  transform: rotate(-135deg);
+  width: 52px;
+
+  .triangle-min {
     -moz-transform: rotate(-180deg);
-    -o-transform: rotate(-180deg);
     -ms-transform: rotate(-180deg);
-    border-right: 2px solid $color9;
-    border-bottom: 2px solid $color9;
+    -o-transform: rotate(-180deg);
```
Comment (clementprdhomme) [inline app/styles/components/c-box-triangle.scss]: Same here: remove the autoprefixed rules.

File: app/styles/components/c-info-map-content.scss [hunk 3]
```diff
@@ -1,36 +1,35 @@
...
 .c_info_map_content {
   width: 360px;
   h2 {
     font-size: 14px;
     font-weight: $font-weight-bold;
     letter-spacing: -0.1px;
-    color: $color9;
-    &:after {
+    color: $color-4;
+    &::after {
```
Comment (clementprdhomme) [inline app/styles/components/c-info-map-content.scss]: You don't need to add the two colons here.

File: app/styles/components/c-menu.scss [hunk 4]
```diff
@@ -1,142 +1,169 @@
...
+.c-menu {
   img {
     height: 70px;
     margin: 0 10px 0 0;
   }
-  height: 80px;
+  align-items: center;
   display: flex;
-  text-transform: capitalize;
+  font-size: 16px;
```
Comment (clementprdhomme) [inline app/styles/components/c-menu.scss]: Use a variable for:
* `font-size`
* `font-weight`

File: app/styles/components/c-menu.scss [hunk 5]
```diff
@@ -1,142 +1,169 @@
...
-      position: absolute;
-      width: 200px;
-      background-color: $color9;
-      z-index: 99;
       padding: 15px;
+      position: absolute;
+      transform: translate(-40%, 0);
       width: 160px;
-      transform: translate(-40%, 0px);
+      z-index: 99;
```
Comment (clementprdhomme) [inline app/styles/components/c-menu.scss]: This should probably be standardized. See if you can use a round number. Take care of possible issues with the `z-index` of other components such as the modal for example.

File: app/styles/components/c-menu.scss [hunk 6]
```diff
@@ -1,142 +1,169 @@
...
+
       li {
         display: block;
-        padding-left: 0 !important;
-        margin-right: 0 !important;
         line-height: 40px;
+        margin-right: 0 !important;
```
Comment (clementprdhomme) [inline app/styles/components/c-menu.scss]: Do we really need the `!important` tag?

File: app/styles/components/c-menu.scss [hunk 7]
```diff
@@ -1,142 +1,169 @@
...
+        padding-left: 0 !important;
+
         a {
-          opacity: 0.9;
+          color: #0d1a34;
```
Comment (clementprdhomme) [inline app/styles/components/c-menu.scss]: Use a variable for:
* `color`
* `font-size`
* `font-weight`

File: app/styles/components/c-slider-case.scss [hunk 8]
```diff
@@ -1,55 +1,62 @@
...
+@import '../_settings.scss';
 
-.c_slider_case {
-  font-family: $font-family-primary;
-  display: flex !important;
-  justify-content: center;
+.c-slider-case {
   align-items: center;
-  padding-top: 90px;
+  display: flex !important;
```
Comment (clementprdhomme) [inline app/styles/components/c-slider-case.scss]: Do we need `!important`? Should be avoided as much as possible.

File: app/styles/components/c-slider-case.scss [hunk 9]
```diff
@@ -1,55 +1,62 @@
...
     display: block;
-    width: 463px;
-    max-width: 463px;
     margin-right: 18px;
+    max-width: 463px;
+    width: 463px;
+
     h2 {
+      color: $color-10;
       font-size: 14px;
```
Comment (clementprdhomme) [inline app/styles/components/c-slider-case.scss]: Use a variable.

**Comment by clementprdhomme** [comment]
Hey Héctor,

I added a few comments on your code, could you take a look?

I'm also having hard times to see what you did in the PR. For example, the case study slider has a weird/broken behavior on Safari. Can you tell me what did you focus on?

If responsiveness was part of it, you should probably take a look at the project with the responsive tools of Google Chrome because even the header menu seems unfinished.

File: app/styles/_settings.scss [hunk 10]
```diff
@@ -1,42 +1,49 @@
-@import url('https://fonts.googleapis.com/css?family=Roboto:300,400,700');
+@import url(https://fonts.googleapis.com/css?family=Roboto:300,400,700);
+@import url(https://fonts.googleapis.com/css?family=Roboto+Slab:400,300,100,700);
 
 // Font families
 $font-family-1: 'Roboto', sans-serif;
+$font-family-2: 'Roboto Slab', serif;
 
 // Font sizes
-$font-size-tiny: 10px;
```
Comment (tiagojsag) [inline app/styles/_settings.scss]: please stop breaking CS

**Pull request closed without merging**